en/option/partial/clip.md

```diff
@@ -12,6 +12,7 @@
 If clip the overflow on the coordinate system. Clip results varies between series:
 
 + Scatter：Ignore the symbols exceeds the coordinate system. Not clip the elements.
++ EffectScatter the symbols exceeds the coordinate system. Not clip the elements.
```

Comment:
It seems there is missing `: Ignore`.
BTW, may these two series be merged as `Scatter/EffectScatter`?

Reply:
> may these two series be merged as Scatter/EffectScatter

Yes, I think that would be better.